### PR TITLE
Update screen sharing end button UI

### PR DESCRIPTION
**Pull request by rasmustautsglia**
The end button for screen sharing is out of date. This PR replaces the icon for that button, changes the padding in the stackView and adjusts the size of the engagement Endbutton according to the design

MOB 1708

### CODE REVIEW

File: GliaWidgets/Component/Button/Action/ActionButton.swift [hunk 1]
```diff
@@ -12,11 +12,12 @@ class ActionButton: UIButton {
     }
 
     private let style: ActionButtonStyle
-    private let kHeight: CGFloat = 40.0
+    private let kHeight: CGFloat
     private let kContentInsets = UIEdgeInsets(top: 6, left: 16, bottom: 6, right: 16)
 
-    init(with style: ActionButtonStyle) {
+    init(with style: ActionButtonStyle, kheight: CGFloat = 40.0) {
```
Comment (gersonnoboa) [inline GliaWidgets/Component/Button/Action/ActionButton.swift:18]: It can just be called `height`

**Review by gersonnoboa — APPROVED** [review]

**Review by igorkravchenko — APPROVED** [review]

**Review by EgorovEI — APPROVED** [review]

File: GliaWidgets/Component/Header/Header.swift [hunk 2]
```diff
@@ -94,7 +94,7 @@ class Header: UIView {
         titleLabel.textAlignment = .center
 
         rightItemContainer.axis = .horizontal
-        rightItemContainer.spacing = 8
+        rightItemContainer.spacing = 16
```
Comment (dukhovnyi) [inline GliaWidgets/Component/Header/Header.swift:97]: will it affect snapshot tests?

**Review by dukhovnyi — APPROVED** [review]

File: GliaWidgets/Component/Header/Header.swift (hunk 2 above)
Comment (rasmustautsglia) [reply GliaWidgets/Component/Header/Header.swift:97]: I don't believe it does

**Comment by rasmustautsglia** [comment]
!squash

**Comment by rasmustautsglia** [comment]
!merge

**Pull request merged**